### PR TITLE
Laikh/update habitat

### PR DESCRIPTION
**Pull request by laikhtewari**
# Description

Updating habitat-sim and adding habitat-lab to Polymetis
Based on discussion [here](https://github.com/facebookresearch/fairo/pull/1262#issuecomment-1176808264)

Fixes # (issue)

[T122948452](https://www.internalfb.com/intern/tasks/?t=122948452)

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] Refactor
- [ ] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Documentation only change
- [ ] Datasets Release
- [ ] Models Release

## Type of requested review

- [x] I want a thorough review of the implementation.
- [ ] I want a high level review. 
- [ ] I want a deep design review.

## Before and After

If this PR introduces a change or fixes a bug, please add before and after screenshots (if this is causes a UX change) or before and after examples(this could be plain text, videos etc ) to demonstrate the change.

If this PR introduces a new feature, please also add examples or screenshots demonstrating the feature.

# Testing

Please describe the tests that you ran to verify your changes and how we can reproduce.

# Checklist:

- [ ] I have performed manual end-to-end testing of the feature in my environment.
- [ ] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have added relevant collaborators to review the PR before merge.
- [ ] [Polymetis only] I ran on hardware (1) all scripts in `tests/scripts`, (2) asv benchmarks.

### CODE REVIEW

File: polymetis/polymetis/environment.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ dependencies:
     - breathe
     - dash
     - grpcio ==1.46.0
-    - habitat-sim=0.2.1=py3.8_bullet_linux_fc7fb11ccec407753a73ab810d1dbb5f57d0f9b9
+    - habitat-sim=0.2.2=py3.8_bullet_linux_011191f65f37587f5a5452a93d840b5684593a00
```
Comment (exhaustin) [inline polymetis/polymetis/environment.yml:37]: Polymetis provides two installation methods: 1) Building locally into an conda env created by this `environment.yml`, or 2) Installing from our prebuilt conda package.

Thus, whenever you update a dependency in the `environment.yml` file, you also need to update the corresponding dependency in our [conda package metadata](https://github.com/facebookresearch/fairo/blob/8cd347337d3f21258a653be0e90d0e369f49def5/polymetis/polymetis/conda/conda_recipe/meta.yaml) to ensure that the two installation methods are aligned.

p.s. Sorry, I know it's not ideal having duplicated dependency specifications in the codebase, but until we implement a better solution this is how we have to do things.

**Review by exhaustin — APPROVED** [review]

**Review by 1heart — APPROVED** [review]
Just make sure tests pass before merging.

**Comment by 1heart** [comment]
Update submodule pending habitat-lab merging [this PR](https://github.com/facebookresearch/habitat-lab/pull/911), then merge.

**Comment by 1heart** [comment]
@laikhtewari looks like this is very close -- can you try debugging this in CircleCI SSH mode and check why importing gym seems to fail?

**Comment by 1heart** [comment]
After some discussion with @vincentpierre, it seems unlikely that this will work before Habitat is conda-installable -- even having habitat-lab on PyPi will not be enough (see #1352 for an attempt). I'm just going to close this for now, and revisit when habitat-lab is added as a conda-installable package.

**Pull request closed without merging**